engine/include/cubos/engine/plugins/tools/entity_selector.hpp

```diff
@@ -0,0 +1,20 @@
+#ifndef CUBOS_ENGINE_PLUGINS_ENTITY_SELECTOR_HPP
+#define CUBOS_ENGINE_PLUGINS_ENTITY_SELECTOR_HPP
+
+#include <cubos/engine/cubos.hpp>
+#include <cubos/core/ecs/entity_manager.hpp>
+
+using namespace cubos::engine;
```

Comment:
Never use a `using namespace` in a header file: users which include this file will have their namespace polluted otherwise.
Its okay to use them in source files though! (.cpp)